ste/xfer-anyToRemote-file.go

```diff
@@ -129,29 +131,16 @@ func AttemptSetBlobTier(jptm IJobPartTransferMgr, blobTier azblob.AccessTierType
 		tierAvailable := BlobTierAllowed(blobTier)
 
 		if tierAvailable {
-			_, err := blobURL.SetTier(ctxWithLatestServiceVersion, blobTier, azblob.LeaseAccessConditions{})
-			if err != nil {
-				// This uses a currently true assumption about the code:
-				// the blobTier passed into this is the destination blob tier, which may be overridden by the user.
-				// If the user overrides the blob tier, S2SSrcBlobTier is not overridden.
-				if jptm.Info().S2SSrcBlobTier == blobTier {
-					jptm.LogTransferInfo(pipeline.LogError, jptm.Info().Source, jptm.Info().Destination, "Failed to replicate blob tier at destination. Try transferring with the flag --s2s-preserve-access-tier=false")
-					s2sAccessTierFailureLogStdout.Do(func() {
-						glcm := common.GetLifecycleMgr()
-						glcm.Info("One or more blobs have failed blob tier replication at the destination. Try transferring with the flag --s2s-preserve-access-tier=false")
-					})
-				}
-
-				// If we know the destination tier is possible, something's wrong and we should error out.
-				if tierSetPossibleFail {
-					jptm.LogTransferInfo(pipeline.LogWarning, jptm.Info().Source, jptm.Info().Destination, "Cannot set destination block blob to the pending access tier ("+string(blobTier)+"), because either the destination account or blob type does not support it. The transfer will still succeed.")
-				} else {
-					jptm.FailActiveSendWithStatus("Setting tier", err, common.ETransferStatus.BlobTierFailure())
-				}
+			if tierSetPossibleFail {
+				jptm.LogTransferInfo(pipeline.LogWarning, jptm.Info().Source, jptm.Info().Destination, "Cannot set destination block blob to the pending access tier ("+string(blobTier)+"), because either the destination account or blob type does not support it. The transfer will still succeed.")
```

Comment:
I'm not sure that this log makes sense anymore, since we didn't perform the set tier operation here.

Reply:
Yeah this needs to be removed. Thanks for pointing.

Reply:
It's still here though? We didn't perform the set tier operation here, so it doesn't make much sense.